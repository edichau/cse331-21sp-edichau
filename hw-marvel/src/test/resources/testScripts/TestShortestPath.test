# Test the shortest path of a hero on a short graph

LoadGraph heroes staffSuperheroes.csv
FindPath Ernst-the-Bicycling-Wizard Notkin-of-the-Superhuman-Beard