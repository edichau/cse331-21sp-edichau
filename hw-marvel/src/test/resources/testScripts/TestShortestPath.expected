# Test the shortest path of a hero on a short graph

loaded graph heroes
path from Ernst-the-Bicycling-Wizard to Notkin-of-the-Superhuman-Beard:
Ernst-the-Bicycling-Wizard to Notkin-of-the-Superhuman-Beard via CSE331